PowerEditor/installer/nativeLang/portuguese.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <NotepadPlus>
-	<Native-Langue name="Portuguese Portugal" filename="portuguese.xml" version="6.6.9">
-		<Menu>
-			<Main>
+    <Native-Langue name="Portuguese Portugal" filename="portuguese.xml" version="6.6.9">
```

Comment:
In this line, is the name attribute supposed to be in the native language or in English?
What should the version number be?